android/build.gradle

```diff
@@ -257,6 +249,7 @@ android {
         versionCode 1
         versionName "1.0"
         buildConfigField("boolean", "IS_NEW_ARCHITECTURE_ENABLED", IS_NEW_ARCHITECTURE_ENABLED.toString())
+        buildConfigField("String", "REANIMATED_VERSION_JAVA", "\"${REANIMATED_VERSION}\"")
```

Comment:
Why do we need additional double quotes in `"\"${REANIMATED_VERSION}\""`?

Reply:
First quotes are for gradle to treat it as a string and another for Java to treat it as a string after it's passed 😵